meinberlin/apps/votes/migrations/0003_voting_token_add_token_hash.py

```diff
@@ -1,52 +1,51 @@
 # Generated by Django 3.2.16 on 2023-01-24 11:01
-
+import argon2 as argon2
```

Comment:
why not just `import argon2`?

Reply:
pycharm did that! :D also I removed argon and moved to sha3 which is much faster and should be fine since the tokens are already high-entropy